animareflection/ui
# Update `Banner` component

## Description

##### Task link: https://trello.com/c/5jpIDBCC/234-banner-patches

1. Changed `whitespace` property.
2. Changed the close `Button` position to prevent overlap issues.
3. Update conditional rendering logic.

## Test Steps

1. Verify banner content wraps at reduced screen widths.
2. Verify the banner variant `closeable` content doesn't overlap.
3. Verify the banner component renders appropriately.
